src/shared-components/DataGridWrapper/DataGridWrapper.spec.tsx

```
@@ -0,0 +1,307 @@
+    expect(screen.queryByRole('status')).toBeNull();
+  });
+
+  test('handles row click', () => {
+    render(<DataGridWrapper {...defaultProps} onRowClick={mockOnRowClick} />);
+    fireEvent.click(screen.getByText('Alice'));
+    expect(mockOnRowClick).toHaveBeenCalledWith(defaultProps.rows[0]);
+  });
+
+  test('renders action column correctly', () => {
+    const actionColumnRenderer = (row: TestRow) => (
+      <button data-testid={`action-${row.id}`}>Action</button>
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)